components/layout/multicol.rs

```diff
@@ -99,9 +99,11 @@ impl Flow for MulticolFlow {
         {
             let column_style = self.block_flow.fragment.style.get_column();
 
-            // `None` is 'normal': "UA-specified length. A value of 1em is suggested."
-            let column_gap = column_style.column_gap.0.unwrap_or_else(||
-                self.block_flow.fragment.style.get_font().font_size);
+            let column_gap = match column_style.column_gap {
+                Either::First(x) => x,
+                Either::Second(_) => self.block_flow.fragment.style.get_font().font_size,
```

Comment:
Nit: Since `Either` type doesn't give us any information about what it holds, we should have a name for the value inside (that could give us some hint), something like,

``` rust
Either::Second(_normal) => ...
```